Makefile

```diff
@@ -0,0 +1,6 @@
+markdownlint: mdl
+
+mdl:
+	mdl --rules \
+		MD002,MD003,MD004,MD005,MD006,MD007,MD008,MD009,MD010,MD011,MD012,MD014,MD018,MD019,MD020,MD021,MD022,MD023,MD024,MD026,MD027,MD028,MD029 \
```

Comment:
What do we skip?

Reply:
Just a couple. I can either work on this here or reiterate again in another PR.
The most problematic will be the line length `MD013 - Line length` as there are many occurrences of its violation.